### PR TITLE
fixes for local development

### PR DESCRIPTION
**Pull request by cjpsycos**

### CODE REVIEW

File: web/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,6 +1,8 @@
+spring.profiles.include=avam-mock
+
 spring.security.user.password=secret
+server.port=8083
```
Comment (michael-wirth) [inline web/src/main/resources/application.properties]: 8083 is already taken by the candiate-service  -> use 8086

File: web/src/main/java/ch/admin/seco/jobs/services/jobadservice/Application.java [hunk 2]
```diff
@@ -19,15 +32,66 @@
...
-    public static void main(String[] args) {
-        SpringApplication.run(Application.class, args);
+        String configServerStatus = env.getProperty("configserver.status");
+        log.info("\n----------------------------------------------------------\n\t" +
+                        "Config Server: \t{}\n----------------------------------------------------------",
+                configServerStatus == null ? "Not found or not setup for this application" : configServerStatus);
     }
 
+    /**
+     * Initializes candidateservice.
```
Comment (cjpsycos) [inline web/src/main/java/ch/admin/seco/jobs/services/jobadservice/Application.java]: candidate? job-ad-service

File: web/src/main/java/ch/admin/seco/jobs/services/jobadservice/Application.java [hunk 3]
```diff
@@ -19,15 +32,66 @@
...
+     * Main method, used to run the application.
+     *
+     * @param args the command line arguments
+     * @throws UnknownHostException if the local host name could not be resolved into an address
+     */
+    public static void main(String[] args) throws UnknownHostException {
+        SpringApplication app = new SpringApplication(Application.class);
+        DefaultProfileUtil.addDefaultProfile(app);
+        Environment env = app.run(args).getEnvironment();
+        String protocol = "http";
```
Comment (cjpsycos) [inline web/src/main/java/ch/admin/seco/jobs/services/jobadservice/Application.java]: please from here on externalize the code to the method displayAppStartInfo(env)

File: infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/DefaultAvamService.java [hunk 4]
```diff
@@ -1,48 +1,58 @@
...
 
     @Override
     public void register(JobAdvertisement jobAdvertisement) {
         output.send(MessageBuilder
-                .withPayload(new RegisterJobAdvertisementMessage(jobAdvertisement))
-                .setHeader("action", "register")
+                .withPayload(jobAdvertisement)
+                .setHeader("event", "JOB_ADVERTISEMENT_INSPECTING")
+                .setHeader("source", "job-ad-service")
+                .setHeader("target", "avam")
```
Comment (cjpsycos) [inline infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/DefaultAvamService.java]: would rename it to targetSystem

File: infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/DefaultAvamService.java [hunk 5]
```diff
@@ -1,48 +1,58 @@
...
     }
 
     @Override
     public void register(JobAdvertisement jobAdvertisement) {
         output.send(MessageBuilder
-                .withPayload(new RegisterJobAdvertisementMessage(jobAdvertisement))
-                .setHeader("action", "register")
+                .withPayload(jobAdvertisement)
+                .setHeader("event", "JOB_ADVERTISEMENT_INSPECTING")
+                .setHeader("source", "job-ad-service")
```
Comment (cjpsycos) [inline infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/DefaultAvamService.java]: would rename it to sourceSystem and define an enum MessageSystem

File: web/src/main/java/ch/admin/seco/jobs/services/jobadservice/Application.java (hunk 2 above)
Comment (michael-wirth) [reply web/src/main/java/ch/admin/seco/jobs/services/jobadservice/Application.java]: ups, fixed

File: web/src/main/java/ch/admin/seco/jobs/services/jobadservice/Application.java (hunk 3 above)
Comment (michael-wirth) [reply web/src/main/java/ch/admin/seco/jobs/services/jobadservice/Application.java]: done

File: infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/DefaultAvamService.java (hunk 5 above)
Comment (michael-wirth) [reply infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/DefaultAvamService.java]: done

File: infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/DefaultAvamService.java (hunk 4 above)
Comment (michael-wirth) [reply infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/DefaultAvamService.java]: done

**Pull request merged**